client/site_config_inspect.go

```diff
@@ -115,6 +115,9 @@ func (cli *VanClient) SiteConfigInspectInNamespace(ctx context.Context, input *c
 			result.Spec.Ingress = cli.GetIngressDefault()
 		}
 	}
+	if ingressAnnotations, ok := siteConfig.Data[SiteConfigIngressAnnotationsKey]; ok {
+		result.Spec.IngressAnnotations = asMap(splitWithEscaping(ingressAnnotations, ',', '\\'))
```

Comment:
Yes, as the formating of annotations is a flat string.

E.g.:
```go
ingressAnnotations := "kubernetes.io/ingress.class=nginx,tag1=abc,tag2=123"
```

When stripping the escape character, this would result in a map of:
- kubernetes.io/ingress.class=nginx
- tag1=abc
- tag2=123

If one does not strip the escape character, this will result in the following annotations:
- kubernetes.io/ingress.class=nginx,
- tag1=abc,
- tag2=123

Additionally, if one explicitly needs a comma in an annotation, it can be escaped using `\\,` to be part of the annotation and not split into separate ones.

Reply:
Sounds fair. After going through it, I believe the original `asMap` under `cmd/skupper/skupper.go` might
need an enhancement task, so that it can escape characters properly, as today it simply splits by `=`.
But that is not a concern for your PR.

Thanks @ThomasK33 !